ragna/source_storages/_embedding.py

```diff
@@ -0,0 +1,16 @@
+from ragna.core import Chunk
+
+from ragna.core._components import Embedding, GenericEmbeddingModel
+
+
+class MiniLML6v2(GenericEmbeddingModel):
```

Comment:
How is this different from the one that comes prepackaged with `chromadb`? IIUC, they are the same: https://github.com/chroma-core/onnx-embedding. We should prefer that, because

1. Installing `chromadb` will also unlock Chroma as source storage
2. Compared to `sentence_transformers` `chromadb` is actually a lightweight dependency.